app/lib/attachment_batch.rb

```diff
@@ -63,8 +63,10 @@ def remove_files
             logger.debug { "Adding #{attachment.path(style)} to batch for deletion" }
             keys << attachment.style_name_as_path(style)
           when :filesystem
-            logger.debug { "Deleting #{attachment.path(style)}" }
-            FileUtils.remove_file(attachment.path(style))
+            if File.exist?(attachment.path(style))
+              logger.debug { "Deleting #{attachment.path(style)}" }
+              FileUtils.remove_file(attachment.path(style))
+            end
```

Comment:
I would just pass `force`:
```suggestion
            logger.debug { "Deleting #{attachment.path(style)}" }
            FileUtils.remove_file(attachment.path(style), true)
```

Reply:
Thanks for comment!
Given `force = true` passed test!